packages/devextreme/js/__internal/ui/popup/m_popup.ts

```diff
@@ -1096,18 +1288,10 @@ class Popup<
       case 'toolbarItems':
       case 'useDefaultToolbarButtons':
       case 'useFlatToolbarButtons': {
-        // @ts-expect-error ts-error
-        // NOTE: Geometry rendering after "toolbarItems" runtime change breaks the popup animation first appereance.
-        // But geometry rendering for options connected to the popup position still should be called.
-        const shouldRenderGeometry = !args.fullName.match(/^toolbarItems((\[\d+\])(\.(options|visible).*)?)?$/);
```

Comment:
So it's not actual anymore, now we always want to rerender everything?

Reply:
not actual anymore